src/module/applications/sheets/character.mjs

```diff
@@ -325,12 +335,48 @@ export default class DrawSteelCharacterSheet extends DrawSteelActorSheet {
     if (item.type === "class") {
       const cls = this.document.system.class;
       if (cls && (cls.dsid !== item.dsid)) {
-        ui.notifications.error("DRAW_STEEL.ADVANCEMENT.WARNING.cannotAddNewClass", { localize: true });
+        ui.notifications.error("DRAW_STEEL.ADVANCEMENT.WARNING.cannotAddNewType", {
```

Comment:
In hindsight, we need to throw an error here. Returning falsy indicates the drop was not handled - which it was.